### ISSUE TITLE
New BSIP: Allow non-LTM accounts to be referrer

### ISSUE THREAD

**Issue by abitmore** [bitshares/bsips#186]
Currently only LTMs can be referrers. This caused a problem that in order to earn money, you need to invest some money first, IMHO this drives away potential marketers E.G. small bloggers.

This BSIP proposes a protocol change to allow non-LTM accounts to refer new accounts. The referral income that a non-LTM referrer earned will be split to herself, her registrar and her referrer. If she upgrades to LTM, her future referral income will be kept by herself.

Risks:
This change will potentially turn the referral program into a MLM scheme, which could be illegal in some jurisdictions (I am not a layer, please correct me if I'm wrong), although every account has the option to upgrade to LTM to stop sharing new referral income to the registrar and referrer.

**Comment by froooze** [comment]
What percentage distribution do you suggest?

**Comment by abitmore** [comment]
> What percentage distribution do you suggest?

We can use current values. E.G. currently `80%` fees paid by non-LTM accounts will go to the referral program, I think it's fair if `80%` of referral income earned by a non-LTM account goes to the referral program as well. The `80%` number is configurable by the committee for new accounts, but won't change for existing accounts. An alternative is to add a new parameter for the committee to decide.

**Comment by abitmore** [comment]
On the opposite, due to market fee sharing feature, I'd expect that referrers prefer their referred accounts to not upgrade to LTM. LTM upgrades is around 100 USD per user only, trading fees (and now can be multi-level) can be much more.

**Comment by froooze** [comment]
> > What percentage distribution do you suggest?
> 
> We can use current values. E.G. currently `80%` fees paid by non-LTM accounts will go to the referral program, I think it's fair if `80%` of referral income earned by a non-LTM account goes to the referral program as well. The `80%` number is configurable by the committee for new accounts, but won't change for existing accounts. An alternative is to add a new parameter for the committee to decide.

We have `80% - Registrar` to distribute:
- referral non-LTM: 20%
- referral to the next LTM: `60% - Registrar`

This gives everyone a 20% discount without a LTM (fee upgrades!), but gives also the LTM the possibility to earn extra `60% - Registrar` from the non-LTM referral system.

**Comment by xiao93** [comment]
support

**Comment by dls-cipher** [comment]
> I think the current referral marketers would feel betrayed by this change, because they make the most money from LTM upgrades (just guessing based on previous forum discussions) and this change takes away one reason for upgrading.

Please, correct me if I'm wrong:

Current LTM referral marketers (that are worth mention) is less than 50.
Total active ones is less than 30.

Blockchain has average of 4500 active accounts during the day. 

@abitmore - This is amazing BSIP and much needed imho, but you would have to convince committee to drastically change the fees between Basic and LTM members, so the incentive for LTM upgrade becomes even better than currently is - with reduced price to ~50$.

**Comment by froooze** [comment]
@ dls-cipher: The current fee for a LTM is fine, because every LTM can still give a discount on the new LTM. Reducing LTM fees makes it also more unattractive to sell new LTM.

**Comment by abitmore** [comment]
@dls-cipher @froooze please be aware that https://github.com/bitshares/committee-tools is the intended place to discussion current fee schedule.

**Comment by froooze** [comment]
Any updates on this? 
This is an important BSIP!

**Comment by abitmore** [comment]
@ryanRfox please assign a BSIP number. Thanks.

**Comment by ryanRfox** [comment]
Assigned BSIP78 @abitmore

**Comment by abitmore** [comment]
@ryanRfox I think 79 is better. If I'm correct, 78 has been taken by #213 .

**Comment by ryanRfox** [comment]
You are correct @abitmore I fixed the title here and updated the readme in #228 (please review)

**Comment by abitmore** [comment]
Thanks.

Lately I found the original idea described in OP has flaws and hard to implement efficiently, so the details is still on hold. I'd like to reserve the BSIP number for a new idea for the referral program (@ryanRfox: up to you to decide). An option is to re-add short-term membership feature with a new design (background: we had annual memberships but it's removed due to flaws in designation and implementation).

**Comment by ryanRfox** [comment]
I feel we should: 
- Leave this BSIP assignment as 79
- Create a new Issue for the redesign of "Short-Term Membership" to assign a new BSIP number (80)
- Consider to update 79 to "Deferred" status with proper documentation in this discussion thread.

**Comment by abitmore** [comment]
Perhaps we can add more status, e.g. drafting, draft completed and etc. This one has no draft yet, so the status `Draft` in the list is not accurate.

**Comment by ryanRfox** [comment]
I'd rather not add more status gates at this time. In case of this BSIP, we should have required the draft text be posted into this Issue discussion. Then we can agree the content is to a point where it is ready for a BSIP number assignment. We typically do a good job on this. Other times we like to reserve a BSIP number if we know a draft is coming soon and related to the previously published one (see STEALTH Phase II, Membership).

**Comment by thul3** [comment]
Non LTM refferer is needed badly .

We should focus on BSIP's first which will bring traffic and assets to bitshares instead of using cores time to improve DEX endlessly

"I think the current referral marketers would feel betrayed by this change, because they make the most money from LTM upgrades (just guessing based on previous forum discussions) and this change takes away one reason for upgrading."

So you demand that DEX won't use the most effective and least costly way to promote DEX ?

**Comment by thul3** [comment]
People won't promote DEX if they need to pay for it.
That's a fact which nobody can change no matter what.

**Comment by pmconrad** [comment]
> So you demand that DEX won't use the most effective and least costly way to promote DEX ?

I demand nothing. I also don't present speculation as fact (as in "most effective and least costly").

I just see that you guys are constantly changing the rules in your favour, regardless of any promises made to business owners or other users. You are destroying all credibility. Not that is much left of it anyway.

**Comment by abitmore** [comment]
> "I think the current referral marketers would feel betrayed by this change, because they make the most money from LTM upgrades (just guessing based on previous forum discussions) and this change takes away one reason for upgrading."

I wanted to comment on this but lately found I've replied earlier. https://github.com/bitshares/bsips/issues/186#issuecomment-517859072 .

BTW some rough numbers here.
![image](https://user-images.githubusercontent.com/9946777/67245361-1354bd80-f45c-11e9-80b7-5783f366af91.png)

**Comment by froooze** [comment]
# Simple non-Membership referral system

##Referral chain
`LTM` -> `a` -> `b` -> `c`

- `LTM` gets (80%-Registrar) from `a`
- `LTM` gets (60%-Registrar) from `b`, and `c`
- `a` gets 20% from `b`
- `b` gets 20% from `c`
- `c` gets nothing

Yes, `c` could set up his own referral account with `d` and save 20% on fees. After this `b` would loose referral income from `c`. On the other side `b` can buy a LTM and get (80%-Registrar) from `b` and (60%-Registrar) from `c`

## Advantages
- referral link/income for every user
- LTM can earn through multiple levels
- LTM upgrade increases current referral income

## Disadvantage
User can save 20% market fees by setting up his own referral account. A small fee upgrade (+10%) could compensate this.

**Comment by abitmore** [comment]
@froooze thanks for your example. One problem about it is that `c` can refer himself and create `d`, `e`, `f` and so on to get the chain long, in the end he mainly uses the last account on the link which generate fees, but his original referrer `b` would probably get nothing by referring him, which kills the purpose of the referral program. For good performance we don't want to encourage long chains, but your idea encourages it in financial sense.

**Comment by froooze** [comment]
@abitmore 
yes, `c` can create `d`, but the creation of `e` or `f` doesn't benefit `c` anymore and is just useless.   

We need to minimize the gaming effects on the referral system, but we can't exclude them completely. 
First step is to introduce **Referral-Vesting**  and **Registration Fee** for non-Membership referrals.  

I see the benefits for a non-Membership referral link higher, than the disadvantages.

**Comment by abitmore** [comment]
Unused **Referral Vesting** objects consume RAM. So I think it's better that users explicitly pay a small fee to enable the feature which will create the vesting objects for them.

**Comment by thul3** [comment]
@pmconrad

Why do you think every exchange out there has a refferal system ?
Because it's the most effective and less costly solution to get traffic to your exchange.
Only users from the biggest refferer from Binance are already making more volume than DEX has.

You refuse bitshares DEX to use that massiv tool ?
You really claim we create that refferal progrom for our own favour  ?

You clearly have an issue recognizing who an enemy is and who is trying to bring growth to bitshares DEX.

Topic :

Non LTM accounts should also get high % of reward else you won't be able to attract people with 20% refferal income.Binance which is already saturated gives 40% to ANYONE and other major exchanges give 50-60%

**Comment by froooze** [comment]
@thul3 
1. the 20% are not to attract new user, but to allow current user to attract more users and build a network
2. the 10-20% are only the start and a jump pad to the LTM
3. the 40-60% are only possible because CEX have mobile nr., e-mail and cookies for verification this is not possible on the DEX

**Comment by thul3** [comment]
@frooze you want to attract big webmasters with big traffic pools you need to give them the best offer you have.
Remember bitshares is competing with other exchanges for that traffic,
second tier reward should be max 10% with the ability to break out and get full reward when buying LTM

**Comment by sschiessl-bcp** [comment]
I would suggest to try bringing in selected webmaster(s) and try the solution already possible now provided through bitshares.eu. Any core change will take a while before being in effec.t

**Comment by grctest** [comment]
I agree with making the referral system available to the public without paying an LTM fee. As others have said, you can start referring users to CEX with no up front cost to the referrer.

We need more users through word of mouth marketing like this.

The remaining LTM benefits should still remain behind the membership fee though.

If it's going to cause bloating of RAM due to allowing every account to do this, then perhaps the referral chain should be as simple as 30% non-LTM user, and 70% to the reserve pool? Rather than trying to lookup complex account referral chains, simply take a huge cut directly for growing the reserve pool & if they want a better deal then they can purchase LTM?

---

    BSIP: 79
    Title: Allow non-LTM Accounts to be Referrer
    Status: Deferred
    Type: Protocol
    Created: 2024-03-10
    Discussion: [Discussion Thread URL]

# Abstract
The current BitShares protocol restricts referral privileges to only Life Time Members (LTMs), limiting the pool of potential referrers. This limitation poses a barrier to entry for individuals who wish to participate in the referral program without investing in LTM status. This BSIP proposes expanding the eligibility criteria for referrers to include non-LTM accounts, thereby encouraging broader participation and potentially increasing user acquisition.

# Motivation
The existing policy of exclusively allowing LTMs to serve as referrers discourages potential marketers, such as small bloggers, from participating in the BitShares ecosystem. By restricting referral privileges to LTMs, the protocol inadvertently creates a financial barrier for those who wish to earn referral income without the upfront investment required for LTM status. This restriction hampers the organic growth of the platform and limits its reach within the wider community.

Additionally, the evolving landscape of affiliate marketing has shifted towards social media platforms such as TikTok, YouTube Shorts, Instagram, and others. These platforms offer immense potential for user acquisition and marketing, yet the current LTM requirement poses a significant barrier to entry for individuals looking to leverage these channels for BitShares promotion.

# Rationale
Expanding the referral program to include non-LTM accounts addresses the aforementioned limitations and promotes greater inclusivity within the BitShares ecosystem. Allowing non-LTM accounts to refer new users not only broadens the pool of potential referrers but also incentivizes grassroots marketing efforts, thereby facilitating user acquisition and platform growth. By removing the LTM requirement for referral participation, BitShares can attract a more diverse array of individuals who may contribute to the platform's development and adoption.

# Specifications
The proposed protocol change entails modifying the referral system to accommodate non-LTM accounts as referrers. Specifically, the referral income earned by non-LTM referrers will be distributed among themselves, their registrars, and their referrers. Upon upgrading to LTM status, a referrer will retain their future referral income.

However, concerns have been raised regarding the potential RAM bloat resulting from tracking the referral earnings of every account. To mitigate this issue, it is proposed that a significant portion, such as 70%, of the referral income earned by non-LTM referrers be directly allocated to the reserve pool. This approach mirrors the revenue-sharing model employed by platforms like Twitch, where creators receive a portion of earnings while the platform retains a substantial cut for operational purposes.

# Discussion
The proposal to allow non-LTM accounts to participate in the referral program has garnered mixed feedback from the community. While many support the idea of broadening referral eligibility to encourage user growth, concerns about RAM usage and the potential for abuse have been raised. The suggestion to allocate a significant portion of non-LTM referrers' earnings to the reserve pool aims to address these concerns by mitigating RAM bloat and ensuring sustainable platform growth.

# Summary for Shareholders
Expanding the referral program to include non-LTM accounts presents an opportunity to foster community engagement, drive user acquisition, and stimulate platform growth. By removing barriers to participation and incentivizing grassroots marketing efforts, BitShares can enhance its competitiveness in the decentralized exchange market and attract a wider user base.
